packages/lambda-powertools-middleware-log-timeout/index.js

```diff
@@ -0,0 +1,42 @@
+const Promise = require('bluebird')
+const Log = require('@perform/lambda-powertools-logger')
+
+module.exports = (thresholdMillis = 10) => {
+  let isTimedOut
+  let promise
+
+  const resetPromise = () => {
+    if (promise) {
+      promise.cancel()
+      promise = undefined
+    }
+  }
+
+  return {
+    before: (handler, next) => {
+      const timeLeft = handler.context.getRemainingTimeInMillis()
+      handler.context.callbackWaitsForEmptyEventLoop = false
```

Comment:
We can't set `callbackWaitsForEmptyEventLoop` in this middleware as it can have undesired side-effects.

Seeing as we can clear the timeout in `after` and `error`, I think we can remove this altogether

Reply:
Yeah, that should work, good catch!